internal/controllers/objectdeployments/new_revision_reconciler.go

```diff
@@ -116,3 +121,10 @@ func (r *newRevisionReconciler) newObjectSetFromDeployment(
 	}
 	return newObjectSet, nil
 }
+
+func latestRevisionNumber(prevObjectSets []genericObjectSet) int64 {
```

Comment:
is this related to the CEL prober?

Reply:
Nope, 2nd commit is a bugfix that fixes a problem due to template schema changing.